lib/tic_tac_toe.rb

```diff
@@ -0,0 +1,5 @@
+class TicTacToe
+  def initialize
+    puts "Welcome to a game of Tic-Tac-Toe!"
```

Comment:
So there are a few reasons to avoid performing real work in your constructors which we will get to talk about, but a very simple one is context. Every class has a `new` method and the expectation is that it will instantiate the class. We want this behavior to be consistent and reproducible since the method name is consistent across classes in Ruby. We're going to end up talking about naming a lot :)

When we call `new` here we now have the additional side effect of an output being printed when the expectation is simple instantiation. In order to avoid this conflict of expectations we can move this behavior into a class method that is named to provide context into what will occur when the method is invoked.

Reply:
Makes sense, I changed it to an instance method